src/utils.ts

```diff
@@ -80,3 +80,28 @@ export function assign(target: object, ...args: Array<object>) {
   });
   return target;
 }
+
+/**
+ * Omits properties from an object
+ *
+ * @private
+ * @param {object} target - Target object
+ * @param {Array<string>} paths - Paths to be ommited
+ * @returns {object} A new object without paths speciefied in paths argument
+ */
+export function omit(target: object, paths: Array<string>) {
+  if (!paths.length) {
+    return target;
+  }
+
+  return Object
+    .keys(target)
+    .filter((key) => paths.indexOf(key) < 0)
+    .reduce((acc, val) => (
+      target.hasOwnProperty(val)
```

Comment:
Is this repeated use of `assign` fast enough? Wouldn't it be faster to create a clone of the object once, and then delete the omitted properties?